### PR TITLE
html-css-popup

### PR DESCRIPTION
**Pull request by OlStani**
### HTML CSS Popup

[Demo](https://olstani.github.io/popup/) |
[Code base](https://github.com/OlStani/popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/OlStaniev/html-css-popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
+@charset "UTF-8";
```
Comment (artem-trubin) [inline submissions/OlStaniev/html-css-popup/css/style.css:1]: Don't add autogenerated files into the homeworks folder. Only files you yourself typed in.

File: submissions/OlStaniev/html-css-popup/css/style.scss [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-size: inherit;
+    font-weight: 400;
+}
+
+/*--------------------*/
```
Comment (artem-trubin) [inline submissions/OlStaniev/html-css-popup/css/style.scss:92]: If you want to have dividers of some sort, it is also nice to name them. Like
```css
/*----------Header----------*/
```
or something like that.

File: submissions/OlStaniev/html-css-popup/css/style.scss [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        justify-content: space-between;
+        align-items: center;
+        padding: 5rem;
+
+        &_logo {
+            font-size: 70px;
+            font-weight: bold;
+        }
+    }
+
```
Comment (artem-trubin) [inline submissions/OlStaniev/html-css-popup/css/style.scss:119]: Empty line.

File: submissions/OlStaniev/html-css-popup/css/style.scss [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+        border-radius: 20px;
+        margin: 50px auto;
+        padding: 20px;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        max-height: 70vh;
+        overflow: hidden;
+        overflow-y: auto;
+
```
Comment (artem-trubin) [inline submissions/OlStaniev/html-css-popup/css/style.scss:141]: Empty line.

File: submissions/OlStaniev/html-css-popup/css/style.scss [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+    }
+
+}
+
+.main {
+    width: 100%;
+    margin: 0 auto;
+    position: relative;
+
+
```
Comment (artem-trubin) [inline submissions/OlStaniev/html-css-popup/css/style.scss:127]: Two empty lines. It is fine to leave an empty line between two blocks, but be consistant, you have only one line between other blocks.

File: submissions/OlStaniev/html-css-popup/index.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="header__wrapper">
+            <div class="header__wrapper_logo">FOOD DELIVERY</div>
+            <div class="header__wrapper_checkbox headerInput">
+                <label for="drop">Choose Delivery</label>
+
```
Comment (artem-trubin) [inline submissions/OlStaniev/html-css-popup/index.html:21]: Empty line.

File: submissions/OlStaniev/html-css-popup/index.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+        </div>
+    </header>
+    <input type="checkbox" id="drop" class="inputHeader">
+    <span class="spanHeader"></span>
+    <main class="main">
+        <div class="main__wrapper">
+            <div class="main__wrapper_item mainList">
+                <ul class="mainList__list">
+                    <li class="mainList__list_item listItem"><a class="imageAnchor" href="#"><img
+                                src="./img/mcdonalds.svg" alt="mcdonalds"></a></li>
```
Comment (artem-trubin) [inline submissions/OlStaniev/html-css-popup/index.html:32]: `alt` attribute for navigation elements, like icons, should be left empty.

**Review by artem-trubin — COMMENTED** [review]
Nice custom design. Let's see how you can improve your work.

**Comment by Roophee** [comment]
Good work!
Not bad  design but you can make it better with responsive styles for mobile devices.
This up to you, but it will be look more perfect and complete  with the responsive styles.

**Comment by artem-trubin** [comment]
Nice work!

**Comment by OlStani** [comment]
thanks!)

**Pull request merged**